packages/ui/src/vue/components/ContentCode.vue

```diff
@@ -95,12 +141,46 @@ export default {
 
     renderedContext () {
       return decorateContext(this.context)
+    },
+
+    iframeSrc () {
+      const themeId = this.currentTheme.id
+      return `${window.UIengine.base}${this.pathPrefix}/${themeId}/${this.pathPostfix}.html`
     }
   },
 
+  watch: {
+    expanded (oldValue, newValue) {
```

Comment:
Can we make this one async, so we can use await instead of explicitely handling the promise?